stripe/src/main/java/com/stripe/android/paymentsheet/ui/BillingAddressView.kt

```diff
@@ -108,16 +108,19 @@ internal class BillingAddressView @JvmOverloads constructor(
         postalCodeView.inputType = config.inputType
     }
 
-    private val newCountryCallback = { newCountry: Country ->
-        updateStateView(newCountry)
-        updatePostalCodeView(newCountry)
+    private val newCountryCodeCallback = { newCountryCode: CountryCode ->
+        updateStateView(newCountryCode)
+        updatePostalCodeView(newCountryCode)
         _address.value = createAddress()
 
         postalCodeValidator.isValid(
             postalCode = postalCodeView.value.orEmpty(),
-            countryCode = newCountry.code
+            countryCode = newCountryCode.twoLetters
```

Comment:
I have another review that will simplify this postalCodeValidator, so not spending too much time on it here.